joint_teapot/utils/joj3.py

```diff
@@ -201,13 +201,22 @@ def generate_title_and_comment(
         comment += "\n"
         for i, result in enumerate(stage["results"]):
             if not single_case:
-                comment += f"### Case {i} - Score: {result['score']}\n"
+                comment += (
+                    f"<details><summary>Case {i} - Score: {result['score']}</summary>\n"
+                )
             if result["comment"].strip() != "":
                 comment += f"{result['comment']}\n"
+            else:
```

Comment:
We will never have this field for failed or passed in JOJ3. No need to comment failed or passed here, the parser will generate it if needed.